aspnetcore/security/enforcing-ssl.md

```diff
@@ -351,30 +388,79 @@ See [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/6199).
 
 ## Trust HTTPS certificate on Linux
 
-<!-- Instructions to be updated by engineering team after 5.0 RTM. -->
+Establishing trust is browser specific. The following sections provide instructions for the Chromium browsers Edge and Chrome and for Firefox.
+
+### Trust HTTPS certificate on Linux using Edge or Chrome
+
+  * Install the `libnss3-tools` for your distribution.
+  * Create or verify the `$HOME/.pki/nssdb` folder exists on the machine.
+  * Export the certificate with the following command:
+  
+    ```cli
+    dotnet dev-certs https -ep /usr/local/share/ca-certificates/aspnet/https.crt --format PEM
+    ```
+  
+  * Run the following commands:
+  
+    ```cli
+    certutil -d sql:$HOME/.pki/nssdb -A -t "P,," -n localhost -i /usr/local/share/ca-certificates/  aspnet/https.crt
+    certutil -d sql:$HOME/.pki/nssdb -A -t "C,," -n localhost -i /usr/local/share/ca-certificates/  aspnet/https.crt
+    ```
+  
+  * Exit and restart the browser.
+
+### Trust the certificate with Firefox
+
+* Export the certificate with the following command:
+
+  ```vstscli
+  dotnet dev-certs https -ep /usr/local/share/ca-certificates/aspnet/https.crt --format PEM
+  ```
+
+* Create a JSON file at `/usr/lib/firefox/distribution/policies.json` with the following contents:
+
+  ```json
+  {
+      "policies": {
+          "Certificates": {
+              "Install": [
+                  "/usr/local/share/ca-certificates/aspnet/https.crt"
+              ]
+          }
+      }
+  }
+  ```
+
+## Ubuntu trust the certificate for service-to-service communication
```

Comment:
It might be worth putting this section first and making the path comment for other distros here

Reply:
Can you verify I put this is the right place?